system/classes/Kohana/Core.php

```diff
@@ -942,7 +939,7 @@ public static function error_handler($code, $error, $file = NULL, $line = NULL)
 		{
 			// This error is not suppressed by current error reporting settings
 			// Convert the error into an ErrorException
-			throw new ErrorException($error, $code, 0, $file, $line);
+			throw new ErrorException($error, $code, $code, $file, $line);
```

Comment:
Why should the $code and $severity parameters be the same?
http://php.net/manual/en/errorexception.construct.php

Reply:
> While the severity can be any integer value, it is intended that the error constants be used. 

http://us2.php.net/manual/en/errorexception.construct.php

Reply:
?

Reply:
@bkosun see  example by php team: severity = code, code = 0, i dont set code = 0 because its can break users code.

Reply:
Unfortunately I did not find this note on the page indicated.

Reply:
>  severity The severity level of the exception.
Note: While the severity can be any integer value, it is intended that the error constants be used.

Reply:
@bkosun http://us2.php.net/manual/en/class.errorexception.php

Reply:
But why do you think `$code` should be equal to `$severity?`

You yourself quoted:
> Note: While the severity can be any integer value, it is intended that the error constants be used.

These constants are:
http://php.net/manual/en/errorfunc.constants.php#errorfunc.constants

The `$code` parameter can contain any value so that you can correctly handle the error.  

For example, if you have a child `Exception` class that handles a MySQL database error, then `$code` may be a MySQL error code.

Reply:
@bkosun  >  Is not it? 

nope., because severity added in ErrorException.